docs/developer_guide/CI.md

```diff
@@ -0,0 +1,40 @@
+# The CI
+
+Historically, this project has been initiated using our own GitLab instance,
+hence, we designed a GitLab-CI-based continuous integration (CI) workflow.
+When we decided to open-source this project and move it to GitHub,
+we lost our CI.
```

Comment:
I don't think this paragraph is interesting for the documentation. Why would people care?